packages/nextra-theme-docs/src/components/collapse.tsx

```diff
@@ -70,7 +70,7 @@ export function Collapse({
     >
       <div
         ref={innerRef}
-        className="p-2 transform-gpu overflow-hidden transition-opacity duration-500 ease-in-out motion-reduce:transition-none"
+        className="p-2 ltr:pr-0 rtl:pl-0 transform-gpu overflow-hidden transition-opacity duration-500 ease-in-out motion-reduce:transition-none"
```

Comment:
it's better to apply this class only for sidebar folders but not by default, what do you think?

`<Collapse className="ltr:pr-0 rtl:pl-0" ...`

Reply:
Makes sense since this is a general component!